src/browser/documentation/help/alter-user.jsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react'
+import ManualLink from 'browser-components/ManualLink'
+import AdminOnSystemDb from './partials/admin-on-systemdb'
+const title = 'ALTER USER'
+const subtitle = 'Modify a user'
+const category = 'administration'
+const content = (
+  <>
+    <p>
+      The command <code>ALTER USER</code> can be used to modify an existing
```

Comment:
Put `command` after `<code>ALTER USER</code>` since it seems all the others created write it this way?

Reply:
Yep, you're right. Fixed it!